### PR TITLE
Feature/our work page

### PR DESCRIPTION
**Pull request by DeveloperKhan**
### Complete "Our Work" page.

Added several new components:
- Spacer (cause everything was too close together)
- Accent (them little brackets)
- Dot Shadow (using a repeatable Ben-Day svg)
- Chapter Feature (displaying a specific chapter)
- Image Highlight (wraps an image with a dot shadow and Ben-Day)
  - **Note:** Image highlight requires a fixed width and height image rn. There is too much going on between swapping images and resizing to make it dynamic without a lot more effort.

### CODE REVIEW

File: components/Container/index.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Spacer from '../Spacer';
```
Comment (ghost) [inline components/Container/index.js]: Hm, let's try and avoid spacers. It's a bit odd from a semantic standpoint, and I think `section` elements should have a bottom margin by default anyways (worked well on Bits of Good with very few overrides)

File: components/ImageHighlight/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import PropType from 'prop-types';
+import DotShadow from '../DotShadow';
+import Accent from '../Accent';
+import styles from './styles.module.scss';
+
+function ImageHighlight({ children, height, width }) {
+  console.log(height);
```
Comment (ghost) [inline components/ImageHighlight/index.js]: Remove `console.log`

File: components/ImageHighlight/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.accent_2 {
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: rotate(180deg) !important;
```
Comment (ghost) [inline components/ImageHighlight/styles.module.scss:28]: The `!important` flag isn't necessary

File: components/Project/item-interface.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const project = {
```
Comment (ghost) [inline components/Project/item-interface.js:3]: Good refactor! 👍

File: components/ChapterTable/styles.module.scss [hunk 5]
```diff
@@ -10,20 +10,24 @@
...
   .chapter_select:first-child {
     margin-left: 0;
   }
 }
 
 .chapter_select {
-  margin: 12px;
+  margin: 12px 12px 0 12px;
+  background-color: transparent;
+  border: none;
```
Comment (ghost) [inline components/ChapterTable/styles.module.scss]: We could elevate these `border` and `background` resets to a global style for all `button` elements

**Review by ghost — CHANGES_REQUESTED** [review]
Just some small changes but looks great overall!

File: components/ImageHighlight/styles.module.scss (hunk 3 above)
Comment (DeveloperKhan) [reply components/ImageHighlight/styles.module.scss:28]: !important flag was necessary. It interacted poorly with the scale transform in the Accent file without it

Comment (DeveloperKhan) [reply components/ImageHighlight/styles.module.scss:28]: e.i. it wouldn't rotate

**Comment by abhisuri97** [comment]
Hello, totally not involved here, but one thing I noticed was that there's a link field for projects. Keep in mind that some projects may not be live and sometimes nonprofits may abandon projects once they get funding to get a formal dev agency to do those projects. It might be worth looking into an option to give the option to instead specify a link to code in the case the project link cannot be shared. (perhaps just add a prop type in addition to link that is called 'linkType' and has the value 'Project' or 'Code')

File: components/Container/index.js (hunk 1 above)
Comment (DeveloperKhan) [reply components/Container/index.js]: fixed

File: components/ImageHighlight/index.js (hunk 2 above)
Comment (DeveloperKhan) [reply components/ImageHighlight/index.js]: fixed

File: components/ChapterTable/styles.module.scss (hunk 5 above)
Comment (DeveloperKhan) [reply components/ChapterTable/styles.module.scss]: fixed

**Comment by abhisuri97** [comment]
Thank you for implementing that!

**Review by ghost — APPROVED** [review]

**Pull request merged**